### PR TITLE
Feature/tg 86 create program

### PR DESCRIPTION
**Pull request by Adrian-Mendoza-Jala**
## Summary
- [TG-86: Create file for colors](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/c9f3159b409fd920f138366b513c8153b6f03d20)

- [TG-86: Fix issue](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/fea28218554b766472044488f808d534bac1921b)

- [TG-86: Update service for program](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/40489461790f8ea3f0a472a4011fc78e5638cf90)

- [TG-86: Refactor Styles for Sidebar and Navbar](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/2f50fc255d8e45a415c1e1950df7c9c55ee8accf)

- [TG-86: Refactor Custom Button](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/601026993fcbb2bd4db75fe04e5aab8dffca6e6f)

- [TG-86: Update components for program form](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/7cedcea7a390edc63d44638f810e114037499919)

- [TG-86: Add Create Form](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/6b502400d8ebd32fea25ac5d6e07073079aa28b3)

- [TG-86: Fix Merge conflicts and Issues](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/commit/f77d30e0240b8180dba4eb38cae8dc88e8afb0ee)

## Images
- Programs
image

- Button
image

- Form
image

- Validation
image

image

- Response
image

- Success Message
image

- Course Button
image

## Reviewers
@AndresJavierNazzari @carlos-canedo @CrisCastroI

### CODE REVIEW

**Review by AJN-jalaU — APPROVED** [review]

File: src/app/modules/app/pages/programs/programs.component.ts [hunk 1]
```diff
@@ -25,7 +30,7 @@ export class ProgramsComponent {
   constructor(
     public programService: ProgramService,
     private snackBar: MatSnackBar,
-    private readonly dialogService: DialogService
+    public dialogService: DialogService
```
Comment (carlos-canedo-dev) [inline src/app/modules/app/pages/programs/programs.component.ts]: Public services must be readonly

File: src/styles/variables.sass
Comment (carlos-canedo-dev) [inline src/styles/variables.sass:1]: This global variables should be declared according to the angular materials' theming standard.

Comment (Adrian-Mendoza-Jala) [reply src/styles/variables.sass:1]: I will put this as TODO, because the edit course needs some things from this implementation

Comment (carlos-canedo-dev) [reply src/styles/variables.sass:1]: Yup, I think you could create a new US to do the whole Angular Material Theming.

**Review by CrisCastroI — APPROVED** [review]

**Review by carlos-canedo-dev — APPROVED** [review]

**Pull request merged**